### PR TITLE
Fix JSON resteasy reactive libraries

### PR DESCRIPTION
**Pull request by loicmathieu**

### CODE REVIEW

**Review by geoand — APPROVED** [review]
Thanks!

File: docs/src/main/asciidoc/resteasy-reactive.adoc [hunk 1]
```diff
@@ -680,16 +680,16 @@ public class Endpoint {
...
 .Table Context object
 |===
 |GAV|Usage
 
-|`io.quarkus:quarkus-resteasy-jackson`
+|`io.quarkus:quarkus-resteasy-reactive-jackson`
 |https://github.com/FasterXML/jackson[Jackson support]
 
-|`io.quarkus:quarkus-resteasy-jsonb`
+|`io.quarkus:quarkus-resteasy-reactive-jsonb<`
```
Comment (gastaldi) [inline docs/src/main/asciidoc/resteasy-reactive.adoc:692]:
```suggestion
|`io.quarkus:quarkus-resteasy-reactive-jsonb`
```

Comment (loicmathieu) [reply docs/src/main/asciidoc/resteasy-reactive.adoc]: Sorry, fixed

**Pull request merged**